frontend/packages/console-shared/src/components/health-checks/health-checks-alert.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
```

Comment:
Why is this file named in `kebab-case`? We name our component files in `PascalCase`

Reply:
My bad. Rename it in ```PascalCase``` formate.